### ISSUE TITLE
Hex colors?

### ISSUE THREAD

**Issue by jessemillar** [charmbracelet/glamour#66]
Some portions of the default style JSON files use hex colors and other parts use 256-style colors. The style docs don't mention anything about non-256 colors, but I'd assume they're meant to work since they're present in the `dark`/`light` styles. Looking through the history of the project, it seems like there was originally no support for hex colors, then it was added in a way that converted hex to 256, then the default became TrueColor...?

Long story short, I can't get custom hex colors to function (e.g. changing `heading.color` in `dark.json` to something like `#ff0000` has no effect). Is this meant to work at this point?

I'm using `glow`/`glow -s dark.json` to test theme changes. I'd ask in the Glow repository but it seems like Glow style questions are often redirected here.

**Comment by muesli** [comment]
Yes, hex color definitions are supported! It will however also depend on your terminal having true color support. If that's not the case, colors will be approximated to closest available color your terminal offers.

Do you have an example for your style definition and markdown content you experiment with?

**Comment by jessemillar** [comment]
My terminal does have true color support. The problem I’m hitting is that hex colors don’t render at all, they just show up as white.

[This is the style file I’m using.](https://github.com/jessemillar/nord-glamour-theme/blob/1f0f65c126369cfef6ce6a0da8dca50b261cdb42/nord.json)

![06AA2AEC-9186-4D4B-B468-2DDD324B421B](https://user-images.githubusercontent.com/4627837/81874706-0efd0880-9544-11ea-96b6-0e1062155aa1.jpeg)

**Comment by muesli** [comment]
Oh, I think i know what's going on: is this the latest stable release v0.2.0 of glow? That does _not_ come with true-color support yet :smiley:

v0.3.0 is pretty much ready for a release, though, and should be out this week!

In the meantime, if you manually build glow from git, your stylesheet seems to work just fine:
![image](https://user-images.githubusercontent.com/146378/81877033-5c996580-9584-11ea-80fa-34314f05b199.png)

**Comment by jessemillar** [comment]
Ah. That would explain things for sure. I installed via `brew` on Linux which is v0.2.0.

What’s the version of Glamour that introduces true color? I plan on making a PR to the GitHub CLI project to allow for custom themes and need to make sure they’re using at least that version.

**Comment by muesli** [comment]
Both `glow` and `glamour` will see a new release here: v0.2.0 for glamour, v0.3.0 for glow. I think you managed to figure out the correct commit indeed! Closing as such.
